src/kOS/Binding/FlightControlManager.cs

```diff
@@ -156,6 +160,88 @@ private bool VesselIsValid(Vessel vessel)
             return vessel != null && vessel.rootPart != null;
         }
 
+        public void SelectAutopilotMode(object autopilotMode)
+        {
+            if (autopilotMode is Direction)
+            {
+                //TODO: implment use of direction subclasses.
+            }
+            else SelectAutopilotMode((string)autopilotMode);
+        }
+
+        public void SelectAutopilotMode(VesselAutopilot.AutopilotMode autopilotMode)
+        {
+            if (currentVessel.Autopilot.Mode != autopilotMode)
+            {
+                if (!currentVessel.Autopilot.CanSetMode(autopilotMode))
+                {
+                    // throw an exception if the mode is not available
+                    throw new kOS.Safe.Exceptions.KOSException(
+                        string.Format("Cannot set autopilot value, pilot/probe does not support {0}, or there is no node/target", autopilotMode));
+                }
+                currentVessel.Autopilot.SetMode(autopilotMode);
+                //currentVessel.Autopilot.Enable();
+                // change the autopilot indicator
+                ((kOS.Module.kOSProcessor)Shared.Processor).SetSASUI((int)autopilotMode);
+                if (RemoteTechHook.IsAvailable(currentVessel.id))
+                {
+                    Debug.Log(string.Format("kOS: Adding RemoteTechPilot: autopilot For : " + currentVessel.id));
+                    // TODO: figure out how to make RemoteTech allow the built in autopilot control.  This may require modification to RemoteTech itself.
```

Comment:
After looking at the code, I'm not even sure it will be possible to fix this in RemoteTech.  Not without known buy-in on all other autopilot enabled mods.  But I only gave it about 2 hours of research.